code/modules/paperwork/paper.dm

```diff
@@ -394,10 +395,8 @@
 			interact(user, readonly = FALSE)
 		return TRUE
 
-	else if(istype(P, /obj/item/stamp) || istype(P, /obj/item/clothing/gloves/ring/seal))
-		apply_custom_stamp(
-			image('icons/obj/bureaucracy.dmi', icon_state = "paper_[P.icon_state]", pixel_x = rand(-2, 2), pixel_y = rand(-2, 2)),
-			"with \the [P]")
+	else if(P.item_flags & ITEM_FLAG_STAMP)
```

Comment:
not super jazzed about using an entire item flag for this given that we're over 3/4 through our supply of them, i would much rather have like. a tool archetype a la pens, or an extension a la cells. feel like maybe we should establish what flags should be used for (things that are common or general enough that they will be useful for a large number of items) and what they shouldn't (anything that has niche functionality that could be pulled out into an extension, anything that could use per-item behavior changes, etc).

that said, this works for now, i understand if you'd rather not bother rewriting it again given all that's on your plate with the wizard rewrites

Reply:
i don't super like that to be honest, but it's whatever, i'll write an extension later